### PR TITLE
Replace Logger declarations

### PR DESCRIPTION
**Pull request by msladek**
https://synjira.atlassian.net/browse/CELDEV-454

### CODE REVIEW

File: celements-model/src/main/java/com/celements/filebase/AttachmentService.java [hunk 1]
```diff
@@ -72,7 +72,7 @@
 @Component
 public class AttachmentService implements IAttachmentServiceRole {
 
-  private static Logger _LOGGER = LoggerFactory.getLogger(AttachmentService.class);
+  private static Logger LOGGER = LoggerFactory.getLogger(AttachmentService.class);
```
Comment (fpichler) [inline celements-model/src/main/java/com/celements/filebase/AttachmentService.java]:
```suggestion
  private static final Logger LOGGER = LoggerFactory.getLogger(AttachmentService.class);
```

File: LoadTesting/src/main/java/com/celements/web/utils/DocumentCreationWorker.java [hunk 2]
```diff
@@ -114,12 +114,12 @@ protected void runInternal() {
         context.getWiki().saveDocument(doc, context);
         docsCreated++;
       } catch (XWikiException e) {
```
Comment (fpichler) [inline LoadTesting/src/main/java/com/celements/web/utils/DocumentCreationWorker.java]:
```suggestion
      } catch (XWikiException | NullPointerException exp) {
```

File: LoadTesting/src/main/java/com/celements/web/utils/DocumentCreationWorker.java [hunk 3]
```diff
@@ -114,12 +114,12 @@ protected void runInternal() {
...
-        mLogger.error(context.getWiki() + " - " + docFullName, e);
+        LOGGER.error(context.getWiki() + " - " + docFullName, e);
```
Comment (fpichler) [inline LoadTesting/src/main/java/com/celements/web/utils/DocumentCreationWorker.java]:
```suggestion
        LOGGER.error("{} - {}", context.getWiki(), docFullName, exp);
```

File: LoadTesting/src/main/java/com/celements/web/utils/DocumentCreationWorker.java [hunk 4]
```diff
@@ -114,12 +114,12 @@ protected void runInternal() {
...
       } catch (NullPointerException npe) {
```
Comment (fpichler) [inline LoadTesting/src/main/java/com/celements/web/utils/DocumentCreationWorker.java]:
```suggestion
```

File: LoadTesting/src/main/java/com/celements/web/utils/DocumentCreationWorker.java [hunk 5]
```diff
@@ -114,12 +114,12 @@ protected void runInternal() {
...
-        mLogger.error(context.getWiki() + " - " + docFullName, npe);
+        LOGGER.error(context.getWiki() + " - " + docFullName, npe);
```
Comment (fpichler) [inline LoadTesting/src/main/java/com/celements/web/utils/DocumentCreationWorker.java]:
```suggestion
```

**Review by fpichler — CHANGES_REQUESTED** [review]

**Comment by msladek** [comment]
@fpichler please note that I only performed workspace wide search&replace operations. it isn't the original intention of these PRs to refactor anything else except the logger declarations and definitons in order to get rid of commons-logging.

some formattings may have occured automatically on file save according to our formatting rules.

**Review by fpichler — APPROVED** [review]

**Pull request merged**